ssl_cert_tracker/models.py

```diff
@@ -13,63 +13,93 @@
                 obj.save()
             else: # this is either update/insert
-                md5_count = TestCertsHistory.objects.filter(orion_id=self.orion_id).filter(md5=self.md5).count()
+                md5_count = NmapHistory.objects.filter(orion_id=self.orion_id).filter(md5=self.md5).count()
                 if md5_count == 0:
-                    print ("We are doing insert....")
-                    obj = TestCertsHistory(cert_id=int(pk_val), \
+                    obj = NmapHistory(cert_id=int(pk_val), \
                                            orion_id=self.orion_id, \
                                            md5=self.md5, \
                                            status="changed", \
                                            xml_data=self.xml_data)
                     obj.save()
                 else:
-                    print ("Certificate found.....We are doing update....")
-                    TestCertsHistory.objects.filter(orion_id=self.orion_id).update(status="found")
+                    NmapHistory.objects.filter(orion_id=self.orion_id).update(status="found")
 
     def save(self, *args, **kwargs):
-        print("We are in Model.save")
-        super(TestCertsData,self).save(*args, **kwargs)
+        super(NmapCertsData, self).save(*args, **kwargs)
         pk_val = self.pk
-        print(pk_val)
-
         self.update_cert_history(pk_val)
+    
+    def get_cert_handle(o_id):
+        """get_cert_handle """
+        return NmapCertsData.objects.get(orion_id = o_id)
+
+    def get_cert_state(o_id, hash_md5):
+        """orion_id_exist """
+        if NmapCertsData.objects.filter(orion_id=o_id).count() == 0:
+            return_code = 1 # new reord
+        elif NmapCertsData.objects.filter(orion_id=o_id).filter(md5=hash_md5).count() == 0:
+            return_code = 2 # cert changed
+        else: # cert has not changed
+            return_code = 0
+        return return_code
+
+    def __str__(self):
+        """__str__ """
+        return self.orion_id
 
-class TestCertsHistory(models.Model):
-    """TestCertsHistory. Model struture for NMap result diff response. """
+class NmapHistory(models.Model):
+    """NmapHistory. Model struture for NMap result diff response. """
     cert_id = models.IntegerField()
     orion_id = models.CharField(max_length=100, blank=True, null=True)
-    md5 =  models.CharField(max_length=100, blank=True, null=True)
+    md5 = models.CharField(max_length=100, blank=True, null=True)
     status = models.CharField(max_length=100, blank=True, null=True)
     xml_data = models.TextField()
 
+    def __str__(self):
+        return self.cert_id
+
+class NmapCertsScript(BaseModel, models.Model):
+    """NmapCertsScript. Model struture for NMap scripts. https://nmap.org/nsedoc/"""
+    name = models.CharField(max_length=100, blank=True, null=True)
+    command = models.CharField(max_length=100, blank=True, null=True)
+    description = models.TextField()
```

Comment:
description field is not needed. there is a field with a similar role already defined in BaseModel